subsys/net/ip/l2/ieee802154/ieee802154_fragment.c

```diff
@@ -26,8 +26,8 @@
 #include "6lo.h"
 #include "6lo_private.h"
 
-#define FRAG_REASSEMBLY_TIMEOUT (MSEC_PER_SEC * \
-				 CONFIG_NET_L2_IEEE802154_REASSEMBLY_TIMEOUT)
+#define FRAG_REASSEMBLY_TIMEOUT \
+	K_SECONDS(CONFIG_NET_L2_IEEE802154_REASSEMBLY_TIMEOUT)
```

Comment:
Should convert this to msecs for better granularity. In a separate PR apparently.